catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswRecordCollection.java

```diff
@@ -61,6 +64,8 @@
 
     private Resource resource;
 
```

Comment:
Ya, right now it can be immutable. I made it a HashMap in case any additional properties were needed later on, though.